src/main/java/com/conveyal/datatools/manager/jobs/ProcessSingleFeedJob.java

```diff
@@ -180,17 +181,36 @@ public void jobLogic () {
         }
     }
 
+    /**
+     * Once the job is complete, notify subscribers and provide feedback on the job creation status.
+     */
     @Override
     public void jobFinished () {
+        String message;
         if (!status.error) {
             // Note: storing a new feed version in database is handled at completion of the ValidateFeedJob subtask.
             status.completeSuccessfully("New version saved.");
+            if (feedVersion.validationResult.errorCount > 0) {
+                message = String.format("New feed version created for %s between %s - %s. During validation, we found %s issue(s)",
+                    feedSource.name,
+                    feedVersion.validationResult.firstCalendarDate,
+                    feedVersion.validationResult.lastCalendarDate,
+                    feedVersion.validationResult.errorCount);
+            } else {
+                message = String.format("New feed version created for %s. The validation check found no issues with this new dataset!", feedSource.name);
+            }
         } else {
             // Processing did not complete. Depending on which sub-task this occurred in,
             // there may or may not have been a successful load/validation of the feed.
             String errorReason = status.exceptionType != null ? String.format("error due to %s", status.exceptionType) : "unknown error";
             LOG.warn("Error processing version {} because of {}.", feedVersion.id, errorReason);
+            message = String.format("While attempting to process a new feed version for %s, Data Tools encountered an " +
+                "unrecoverable error. More details: %s", feedSource.name, errorReason);
         }
+        // Send notification to those subscribed to feed version updates.
+        NotifyUsersForSubscriptionJob notifyUsersForSubscriptionJob
+            = new NotifyUsersForSubscriptionJob("feed-updated", feedSource.id, message);
+        notifyUsersForSubscriptionJob.run();
```

Comment:
I think what might give us a more reliable way to test all of this would be to pass the status and feedVersion as arguments to a new subclass of `NotifyUsersForSubscriptionJob` called `NewVersionNotificationJob`. The tests themselves could be modified to check the notification created out of that job for different assertions.

Reply:
Please use previous `NotifyUsersForSubscriptionJob.createNotification` method.

Reply:
Updated.